File: BackEnd/GitHubSimulator.Infrastructure/Models/Issue.cs
using GitHubSimulator.Core.Models.ValueObjects;

namespace GitHubSimulator.Infrastructure.Models;
public class Issue
{
    public Guid Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public DateTime? CreatedAt { get; set; }
    public string AssigneeEmail { get; set; }
    public Guid RepositoryId { get; set; }
    public Guid? MilestoneId { get; set; }
}

File: FrontEnd/GitHubSimulatorUI/src/app/modules/issues/model/Issue.ts
export class Issue {
  id: string | undefined;
  title: string | undefined;
  description: string | undefined;
  createdAt: Date | undefined;
  assignee : {
    email: string | undefined;
  } | undefined;
  repositoryId: string | undefined;
  milestoneId: string | undefined;
}
